google/cloud/pubsub/samples/samples.cc

```diff
@@ -243,10 +242,9 @@ void ListTopicSubscriptions(google::cloud::pubsub::TopicAdminClient client,
                             std::vector<std::string> const& argv) {
   //! [START pubsub_list_topic_subscriptions] [list-topic-subscriptions]
   namespace pubsub = ::google::cloud::pubsub;
-  [](pubsub::TopicAdminClient client, std::string project_id,
-     std::string topic_id) {
-    auto const topic =
-        pubsub::Topic(std::move(project_id), std::move(topic_id));
+  [](pubsub::TopicAdminClient client, std::string const& project_id,
+     std::string const& topic_id) {
+    auto const topic = pubsub::Topic(project_id, topic_id);
```

Comment:
This doesn't seem necessary, but I will assume you changed it for clarity / consistency.

Reply:
Yes, consistency with the surrounding code.